tools/vagrant/README.md

```diff
@@ -5,34 +5,35 @@ The following instructions were tested on Mac OS X El Capitan, Ubuntu 14.04.3 LT
 *Requirements*
 - Install [Vagrant](https://www.vagrantup.com/downloads.html)
  
-### Clone the repository and change directory to `tools/vagrant/custom`
+### Clone the repository and change directory to `tools/vagrant`
 ```
-git clone https://github.com/openwhisk/openwhisk.git
-cd openwhisk/tools/vagrant/custom
+git clone --depth=1 https://github.com/openwhisk/openwhisk.git
+cd openwhisk/tools/vagrant
 ```
 
-### Create VM using custom datastore (Cloudant or CouchDB)
-Follow instructions [tools/db/README.md](../../db/README.md) on how to configure a datastore for OpenWhisk.
+### Create a Vagrant VM
 
-#### Option 1: Create VM using Cloudant DB
+#### Option 1: Create VM using ephemeral CouchDB
+**Important** We advise that you use this method for development of OpenWhisk.
+Please note that no data will persist between two creations of the container.
+For more information on datastore configurations see [tools/db/README.md](../../db/README.md).
 ```
-# Provide credentials for cloudant database with admin permissions
-OW_DB=cloudant OW_DB_USERNAME=xxxxx OW_DB_PASSWORD=yyyyyy vagrant up
+# Configure with couchdb docker container running inside the VM
```

Comment:
This variables are handle inside the Vagrant file and change to upper case, this is worked already done by @lehoanganh  and @domdom82 
It must be "cloudant" lower case and couchdb can be lowercase or uppercase
See Vagrant file https://github.com/csantanapr/openwhisk/blob/96adc9d2b4c968a2e24503162f436776b3a7ed7d/tools/vagrant/Vagrantfile#L66

```
export OW_DB=#{ENV['OW_DB']}
export OW_DB_USERNAME=#{ENV['OW_DB_USERNAME']}
export OW_DB_PASSWORD=#{ENV['OW_DB_PASSWORD']}
if [[ $OW_DB == "cloudant" ]]; then
  export OW_DB="Cloudant"
  export OW_DB_PROTOCOL="https"
  export OW_DB_HOST="$OW_DB_USERNAME.cloudant.com"
  export OW_DB_PORT="443"
else
  export OW_DB="CouchDB"
  export OW_DB_PROTOCOL=#{ENV['OW_DB_PROTOCOL']}
  export OW_DB_HOST=#{ENV['OW_DB_HOST']}
  export OW_DB_PORT=#{ENV['OW_DB_PORT']}
fi
```

I will review all vagrant -> Vagrant